helm/zenko/charts/cloudserver-front/templates/deployment.yaml

```diff
@@ -28,6 +28,16 @@ spec:
               value: "{{- printf "%s-%s" .Release.Name "s3-metadata" | trunc 63 | trimSuffix "-" -}}"
             - name: REDIS_HOST
               value: "{{- printf "%s-%s" .Release.Name "redis" | trunc 63 | trimSuffix "-" -}}"
+{{- if index .Values "mongodb-replicaset" }}
+            - name: S3METADATA
+              value: mongodb
+            - name: MONGODB_HOST
+              value: {{- printf "%s-mongodb-replicaset.%s.svc.cluster.local" .Release.Name .Release.Namespace }}
```

Comment:
`%s-mongodb-replicaset` should do. No need to include the namespace, and `svc.cluster.local` can be deployment-dependent.